printc/__init__.py

```diff
@@ -27,13 +31,16 @@ def printc(*objects, c=15, sep=' ', end='\n', file=stdout, flush=False):
         6 - cyan       14 - brightcyan
         7 - white      15 - brightwhite (default)
     """
-    if not isinstance(c, int):
+    if not isinstance(c, int) or c not in range(16):
         raise ValueError('c has to be a number in range 0..15')
-    msg = sep.join(str(o) for o in objects)
-    if file is stdout and c != 15:
-        if stdout.isatty():
-            msg = add_color_esc_codes(msg, c=c)
-    print(msg, end=end, file=file, flush=flush)
+
+    if all([c != 15,
```

Comment:
calling a function and creating a list of conditions instead of just evaluating
`if ... and ... and ...:` is unnecessary and imho decreases readability